src/game.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
```

Comment:
You should remove the eslint-disable for no-console. Instead of using console.log for debugging, consider using a logger or a debugger.